.github/workflows/python-app.yml

```
@@ -12,8 +12,28 @@ on:
     types: [published]
 
 jobs:
+  version:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+    steps:
+      - name: Checkout main branch
+        uses: actions/checkout@v3
+        with:
+          ref: 'main'
+          #token: ${{ secrets.<ADD_SECRET_HERE> }}
```

I've added an SSH key with permissions scoped to this repository. If I understand correctly this should do it:

```suggestion
          ssh-key: ${{ secrets.SSH_KEY }}
```